pkgs/applications/graphics/write_stylus/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, lib, qt5, makeWrapper, fetchurl, makeDesktopItem }:
+stdenv.mkDerivation rec {
+  name = "write_stylus";
+
+  desktopItem = makeDesktopItem {
+    name = "Write";
+    exec = "Write";
+    comment = "a word processor for hadwriting";
+    desktopName = "Write";
+    genericName = "Write";
+    categories = "Office;Graphics";
+  };
+
+  src = fetchurl {
+    url = "http://www.styluslabs.com/download/write-tgz";
+    sha256 = "1p6glp4vdpwl8hmhypayc4cvs3j9jfmjfhhrgqm2xkgl5bfbv2qd";
+  };
+  sourceRoot = ".";
+  unpackCmd = ''
+    tar xfz "$src"
+    '';
+
+  buildPhase = ":";   # nothing to build
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -R Write $out/
+    # symlink the binary to bin/
+    ln -s $out/Write/Write $out/bin/Write
+
+    mkdir -p $out/share/applications
+    ln -s ${desktopItem}/share/applications/* $out/share/applications/
+  '';
+  preFixup = let
+    libPath = lib.makeLibraryPath [
+      qt5.qtbase        # libQt5PrintSupport.so.5
+      qt5.qtsvg         # libQt5Svg.so.5
+      stdenv.cc.cc.lib  # libstdc++.so.6
+    ];
+  in ''
+    patchelf \
+      --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
+      --set-rpath "${libPath}" \
+      $out/Write/Write
+  '';
+
+  meta = with stdenv.lib; {
```

Comment:
Please mention the license, since Write is unfree.

Reply:
The license does not look like redistributable: 
> The LICENSEE may not reproduce or distribute the SOFTWARE PRODUCT.

so just unfree, I guess.

Reply:
" stdenv.lib.licenses.unfree, "unfree"

    Unfree package that cannot be redistributed. You can build it yourself, but you cannot redistribute the output of the derivation. **Thus it cannot be included in the Nixpkgs channel**. "

So I can close the pull request?

Reply:
You should read "it won't be build by hydra, but compiled locally". There are dozens of unfree derivations in nixpkgs: https://search.nix.gsc.io/?q=unfree&i=nope&files=pkgs&repos=
The formulation is arguably misleading, maybe the docs need to be made clearer.